include/drivers/sensor.h

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright © 2017-2018 The Crust Firmware Authors.
+ * SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+ */
+
+#ifndef DRIVERS_SENSOR_H
+#define DRIVERS_SENSOR_H
+
+#include <dm.h>
+#include <intrusive.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <util.h>
+
+#define SENSOR_OPS(dev) \
+	(&container_of((dev)->drv, struct sensor_driver, drv)->ops)
+
+struct sensor_conversion {
+	const int16_t min_value;  /**< Addition factor. */
+	const int16_t multiplier; /**< Multiplicaton factor. */
+};
+
+struct sensor_info {
+	const char *const              name;    /**< Name exported to SCPI. */
+	uint32_t                       reading; /**< Current value from sensor.
```

Comment:
This should not be here (no code uses it)

Reply:
done